source/crud/tests/unified/estimatedDocumentCount.yml

```diff
@@ -34,6 +34,7 @@ tests:
   - description: "estimatedDocumentCount uses $collStats on 4.9.0 or greater"
     runOnRequirements:
       - minServerVersion: "4.9.0"
+        serverlessMode: forbidServerless
```

Comment:
Ironically, serverless does not support counting via `$collStats`, so these need to be skipped until that has been fixed ([CLOUDP-86318](https://jira.mongodb.org/browse/CLOUDP-86318))